docs/source/clusters/kaizen2/Kaizen-2.md

```diff
@@ -170,6 +170,9 @@ IP Address       Hostname/Description
 128.52.60.35     ocp-staging cluster: API Virtual IP
 128.52.60.36     ocp-staging cluster: Ingress Virtual IP
 
+128.52.60.50     Start: ocp-staging openshift cluster tentant IP
+128.52.60.60     End: ocp-staging openshift cluster tentant IP
```

Comment:
Is this a big enough range if the virtual rose folks need a public ip per virtual machine?

Reply:
this is for the staging cluster, the range for ocp-prod is already in this document and it is:
```
128.52.61.128    Start: OCP-PROD openshift cluster Tenant IP (128.52.61.128/25)
128.52.61.255    End: OCP-PROD openshift cluster Tenant IP (128.52.61.128/25)
```
that's ~128 IPs